site-packages/tasks_ci/toolchain.py

```diff
@@ -48,12 +48,16 @@ def discover_tidy_impl(ext: str) -> str:
     if ret is not None:
         return ret
 
-    for v in ["11", "11.0", "10", "10.0", "12", "12.0"]:
+    for v in ["13", "13.0", "11", "11.0", "10", "10.0", "12", "12.0"]:
         ret = shutil.which("clang-tidy-{}{}".format(v, ext))
         if ret is not None:
             return ret
 
     possible_commands = [
+        "/usr/bin/clang-tidy-13",
+        "/usr/local/bin/clang-tidy-13",
+        "/usr/bin/clang-tidy-12",
+        "/usr/local/bin/clang-tidy-12",
         "/usr/bin/clang-tidy-10",
         "/usr/bin/clang-tidy-11",
         "/usr/local/bin/clang-tidy-10",
```

Comment:
A. In green L22 and L51 the version `12` is checked _last_ (even after `10`). But in L56 - 69 the version `12` is checked after `13`. It is suspicious.

B. In L61 - 64 the lines for version `10` (L61, L63) are interleaved with lines for version `11` (L62, L64). But in L57 - 60 the versions do not interleave. This is suspicious.

C. In L61 - 64 the version `10` is checked ahead of the version `11`. But in lines 22, 51 the version `11` is ahead of `10`. This is suspicious.

Reply:
Version ordering fixed.

Reply:
FYI: In L22 the order is still different (`12` is the last, after `10`).